src/cascadia/TerminalSettingsEditor/MainPage.idl

```diff
@@ -4,7 +4,7 @@
 namespace Microsoft.Terminal.Settings.Editor
 {
     [default_interface]
-    runtimeclass MainPage : Windows.UI.Xaml.Controls.Page
+    runtimeclass MainPage : Windows.UI.Xaml.Controls.Grid
```

Comment:
honestly page should be acceptable -- we may want to make the content property a FrameworkElement, not just a Grid or Control